setup.py

```diff
@@ -51,25 +50,34 @@
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    python_requires=">=3.7",
+    python_requires=">=3.9",
     install_requires=[
         "setuptools",
-        # -*- Extra requirements: -*-
-        "z3c.jbot",
-        "plone.api>=1.8.4",
-        "plone.app.dexterity",
+        "plone.api",
         "plone.restapi",
         "pyotp",
+        "iw.rejectanonymous",
```

Comment:
```suggestion
```
`iw.rejectanonymous` is a soft dependency. My suggestion is to remove it from setup.py and ignore it in the dependency checker.

```
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -121,7 +121,7 @@ Zope = [
   'Products.CMFCore', 'Products.CMFDynamicViewFTI',
 ]
 python-dateutil = ['dateutil']
-
+ignore-packages = ['iw.rejectanonymous']

```

Reply:
@mamico i have took into account your suggestion. i have enable a codecov workflow with upload the summary to codecov.io. can you generate a token on codecov.io and add it to the settings of this package please?

Reply:
I'm sorry, but I don't have access to the settings tab in this collective repository. @jensens, I see you are an owner in the collective organization. Could you possibly help us?

Reply:
I can add a token, but I have no idea what to do at codecov side.

Reply:
@jensens 
> I can add a token, but I have no idea what to do at codecov side.

Nothing is to do, only the token is required to upload the coverage report via gh workflow.

Reply:
@jensens AFAIK you need to open https://codecov.io authenticate using github, search for this repository, or you can probably go directly here: https://app.codecov.io/gh/collective/pas.plugins.tfa/new, now you should see the token for github actions. 
(p.s. I've never used codecov before, so if I'm doing something wrong, please @1letter correct me)

Reply:
`CODECOV_TOKEN` is set. Please try it.